_faq/installation_and_deployment/no-external-ip.md

```diff
@@ -0,0 +1,30 @@
+---
+title: How to deploy Forseti without external IPs?
+order: 1
+---
+{::options auto_ids="false" /}
+Please follow the following steps to install Forseti with internal IP only:
+
+* Prerequisites
+    * Make sure the host subnet has [Google Private IP Access ](https://cloud.google.com/vpc/docs/configure-private-google-access) turned on.
+    * Setup [Cloud NAT](https://cloud.google.com/nat/docs/using-nat) with a bastion host.
+* Install Forseti
```

Comment:
Is it the case that Forseti will be using the VPC network of another project?  Do we need to mention that Forseti should be installed with shared network flags?

Reply:
as per offline discussion, external IP removal does not necessarily involve network change. Henry suggested we leave it to customer's judgement if they want to use another network